core/themes/seven/js/script.js

```diff
@@ -0,0 +1,292 @@
+
+              // Dropdown control gets left aligned
+              $('.expand-dropdown-tabs-control', $tabsWrapper).css('left', 0);
+              $($primaryTabs).css('padding-left', expandControlWidth);
+
+              // Manage classes on wrapper
+              $($tabsWrapper).addClass('responsive-tabs-after').removeClass('desktop-primary-tabs responsive-tabs-before responsive-tabs-mobile');
+            }
+            else {
+              /**
+               * 'mobile' Responsive Tab Strategy
+               */
+              responsiveTabsType = 'mobile';
+
+              // Manage classes on tabs and wrappers
+              $('.duplicated-tab', $primaryTabs).removeClass('duplicated-tab');
+              $($tabsWrapper).addClass('responsive-tabs-mobile').removeClass('responsive-tabs-before responsive-tabs-after desktop-primary-tabs');
+
+              // Figure out how to lay primary tabs behind the expand control
+              var tabsOffset = activeTabNth * (tabHeight + 2);
+              var tabsTopDistance = $($tabsWrapper).position().top;
+              $($primaryTabs).css('top', '-' + tabsOffset + 'px');
+              if (tabsOffset > tabsTopDistance) {
+                expandedTabsHeaderPadding = tabsOffset - tabsTopDistance + (2 * parseInt(defaultHeaderPadding, 10));
+              }
+
+              // Get the active tab's text
+              var $activeTabText = $('<span class="expand-dropdown-tabs-label">' + $('li.active a', $primaryTabs).html() + '</span>');
+              $('.element-invisible', $activeTabText).remove();
+              $('.expand-dropdown-tabs-label', $tabsWrapper).replaceWith($activeTabText);
+              $('.expand-dropdown-tabs-control', $tabsWrapper).css('left', 'auto');
+
+              // Cleanup things that may have been left over from other
+              // responsive tab strategies
+              $('.responsive-tabs-dropdown', $tabsWrapper).remove();
+              $($primaryTabs).css('padding-left', 0);
+            }
+
+            // Insert $responsiveTabsDropdown to markup if it's not empty
+            if ($('li', $responsiveTabsDropdown).length > 0) {
+              $('.duplicated-tab', $responsiveTabsDropdown).removeClass('duplicated-tab');
+              if ($('.responsive-tabs-dropdown').length > 0) {
+                $('.responsive-tabs-dropdown', $tabsWrapper).replaceWith($responsiveTabsDropdown);
+              }
+              else {
+                $($primaryTabs).after($responsiveTabsDropdown);
+              }
+            }
+          }
+        }
+      }
+
+      /**
+       * Check to see when webfont has loaded and adjust the tabs display
+       */
+      var checkFontCounter = 0;
+      // Append an invisible element that will be monospace font or our desired font
+      $('body').append(
+        '<span id="checkFontWrapper" style="font-family: \'Open Sans\', monospace; font-size: 10px; font-weight: 400; position: absolute; z-index: -100; visibility: hidden;">' +
```

Comment:
Could this be done with a class? I recognize the simplicity of understanding the code by putting it all in one place, but this would effectively prevent Open Sans from being replaced with another font in a sub-theme.